docs/Project.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+ObjectFile = "d8793406-e978-5875-9003-1fc021f44a92"
```

Comment:
Please, remove this: as far as I understand, the package itself shouldn't be mentioned in the documentation as dependency, as that would be automatically loaded anyway.  Make sure to update the corresponding `Manifest.toml` as well.

Reply:
Sorry about that, I've updated the files.

Reply:
No worries :wink:  Just for reference, note that [these two lines](https://github.com/abhishalya/ObjectFile.jl/blob/add_docs/.travis.yml#L27-L28) of the Travis script automatically add the local package to the documentation project.

Reply:
Ah, okay. I just thought doing `--project=docs/` would make it act as different project

Reply:
Yes, that option does what you think!  But the command
```julia
using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()
```
says "add the local package to the project", so that you don't need to manually add it to the Project as you did before.  In particular, this makes a difference as it's saying to use the local copy of the package, instead of the last registered version, so that we make sure to provide the documentation for the latest development version of the package.

Reply:
Yep, I understood that now :)